.github/workflows/continuous-delivery.yml

```diff
@@ -54,34 +40,20 @@ jobs:
 
     steps:
       - name: Checkout branch
-        uses: actions/checkout@v4
+        uses: actions/checkout@v4.1.6
 
-      - name: Set up Node.js version
-        uses: actions/setup-node@v4
-        with:
-          node-version-file: ".nvmrc"
-
-      - uses: pnpm/action-setup@v2.2.4
-        name: Install pnpm
-        id: pnpm-install
-        with:
-          run_install: false
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4.0.0
 
-      - name: Get pnpm store directory
-        id: pnpm-cache
-        run: |
-          echo "::set-output name=pnpm_cache_dir::$(pnpm store path)"
-      - uses: actions/cache@v3
-        name: Setup pnpm cache
+      - name: Set up Node.js version
+        uses: actions/setup-node@v4.0.2
         with:
-          path: ${{ steps.pnpm-cache.outputs.pnpm_cache_dir }}
-          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
-          restore-keys: |
-            ${{ runner.os }}-pnpm-store-
+          node-version-file: .nvmrc
+          cache: pnpm
 
       - name: Install dependencies
         run: |
-          pnpm install --frozen-lockfile
+          pnpm install
```

Comment:
@matijs I kind of like having an explicit `--frozen-lockfile` instead of relying on `CI=1` environment variable magic. Any particular reason to remove it?

Reply:
We can add it again. The only reason I removed it was that is was free and in line with the other repos.